bentoml/deployment/serverless/serverless_utils.py

```diff
@@ -40,7 +39,9 @@
 SERVERLESS_BIN_COMMAND = '{}/node_modules/.bin/serverless'.format(BENTOML_HOME)
 
 
-def check_nodejs_comptaible_version():
+def check_nodejs_compatible_version():
+    from bentoml.utils.whichcraft import which
```

Comment:
could you test if the tests pass now without the change? would love to understand why this change was made